libp2p/protocols/relay/client.nim

```diff
@@ -0,0 +1,266 @@
+## Nim-LibP2P
+## Copyright (c) 2022 Status Research & Development GmbH
+## Licensed under either of
+##  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+##  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+## at your option.
+## This file may not be copied, modified, or distributed except according to
+## those terms.
+
+import chronos, chronicles, options
+import ./messages,
+       ./rconn,
+       ./utils,
+       ../../peerinfo,
+       ../../switch,
+       ../../multiaddress,
+       ../../stream/connection
+
+import std/times
+
+logScope:
+  topics = "libp2p circuit relay client"
```

Comment:
Each word in this string is a different topic, so it should be `libp2p relay relay-client` or equivalent